src/components/ObitPane/ObitPane.js

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ObitPane.css';
 
-const ObitPane = () => {
+const ObitPane = ({obit}) => {
+    const [obituary, setObituary] = useState(obit);
+    const [editMode, setEditMode] = useState(false);
```

Comment:
Again, SO CLEVER. I LOVE how reusable this component is. Good job!

Reply:
:)